# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieService.h
# Commit: 3949d9b400d4
# Full Hash: 3949d9b400d41a64102657d2858b09c41de0f142
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-07 21:34:02
# Regressor Bug: 1839918
# File Overlap Count: 3
# Description:
#   Bug 1839918 - Pass BrowsingContext ID and third-party flag through CookieService. r=bvandersloot,cookie-reviewers,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D185303
# ==============================================================================

diff -r 707021ebfd23 -r 3949d9b400d4 netwerk/cookie/CookieService.h
--- a/netwerk/cookie/CookieService.h	Thu Sep 07 08:25:55 2023 +0000
+++ b/netwerk/cookie/CookieService.h	Thu Sep 07 08:25:55 2023 +0000
@@ -99,8 +99,8 @@
 
   bool SetCookiesFromIPC(const nsACString& aBaseDomain,
                          const OriginAttributes& aAttrs, nsIURI* aHostURI,
-                         bool aFromHttp,
-                         const nsTArray<CookieStruct>& aCookies);
+                         bool aFromHttp, const nsTArray<CookieStruct>& aCookies,
+                         uint64_t aBrowsingContextId, bool aIsThirdPartyCookie);
 
  protected:
   virtual ~CookieService();